src/plot/series/custom-svg-series.js

```diff
@@ -99,7 +99,7 @@ class CustomSVGSeries extends AbstractSeries {
     if (animation) {
       return (
         <Animation {...this.props} animatedProps={ANIMATED_SERIES_PROPS}>
-          <CustomSVGSeries {...this.props} animation={null}/>
+          <CustomSVGSeries {...this.props} animation={false}/>
```

Comment:
If animation wasn't false then we would get an infinite stack of animations? I'm not sure if i catch your meaning?